api-module-library/hubspot/manager.js

```diff
@@ -31,8 +31,9 @@ class Manager extends ModuleManager {
             client_id: process.env.HUBSPOT_CLIENT_ID,
             client_secret: process.env.HUBSPOT_CLIENT_SECRET,
             scope: process.env.HUBSPOT_SCOPE,
-            redirect_uri: `${process.env.REDIRECT_URI}/hubspot`,
+            redirect_uri: process.env.REDIRECT_URI,
             delegate: instance,
+            state: JSON.stringify({ app: 'hubspot' })
```

Comment:
I think this one we want to pass in whatever state arg is provided by the caller for getAuthRequirements (this is not quite fully supported in concept yet but it's the logical place to add support) and remove the hardcoded json string here (it's a vestige of how FreshBooks' app was built)

(Then it's on us to pass it in on the get authorize route in our local Frigg application)

Reply:
@seanspeaks The thing is, the `authorizationUri` is set in the [constructor](https://github.com/friggframework/frigg/blob/main/api-module-library/hubspot/api.js#L57) which prevents it from being updated later in the `getAuthRequirements` call. So we can either:
1- update the getAuthUri to read the state on the fly, similar to [slack](https://github.com/friggframework/frigg/blob/main/api-module-library/slack/api.js#L169)
2- accept the api state via the params of getInstance

Thoughts?

Reply:
@Jo-lefthook IMO the best way here is to change the `getAuthUri()` to re-generate the `authorizationUri` with the latest variable updates